deployments/staging/kakarot_deployments.json

```diff
@@ -0,0 +1,58 @@
+{
```

Comment:
are these the real tokens on sepolia?

Reply:
not with the same kakarot, so they are "invisible"